tools/llm_based_sql_translation_validator/run_sql_validator.sql

```diff
@@ -12,26 +12,26 @@ BEGIN
 
 --Sample Input
 /*
-DECLARE var_connection_name STRING DEFAULT 'bq-data-project.us.gemini-sql-validator-connection';
+DECLARE var_connection_name STRING DEFAULT 'poc-env-aks-bq-admin.us.gemini-sql-validator-connection';
 DECLARE var_source_database STRING DEFAULT 'Teradata';
-DECLARE var_source_gcs_path STRING DEFAULT 'gs://gemini-sql-validator/td/*'; 
-DECLARE var_target_gcs_path STRING DEFAULT 'gs://gemini-sql-validator/bq/*';
-DECLARE var_Dataset_name STRING DEFAULT 'gemini_sql_validator';
+DECLARE var_source_gcs_path STRING DEFAULT 'gs://gemini-validator/td/*'; 
+DECLARE var_target_gcs_path STRING DEFAULT 'gs://gemini-validator/bq/*';
 */
 
 DECLARE var_Batch_Id STRING;
 DECLARE var_Insert_Time TIMESTAMP;
 DECLARE var_Model_Name STRING;
 DECLARE Var_Prompt STRING;
 
+SET @@dataset_id ="gemini_sql_validator";
+SET var_Model_Name = "sql-validator-model";
```

Comment:
```suggestion
SET var_model_name = "sql-validator-model";
```
can you please use consistent lower_snake_case? I see several variables in both sql files without proper lower_snake_case, could we fix them?